packages/chains/src/chains/chainMetadata/index.ts

```diff
@@ -160,3 +236,31 @@ export const chains: {
     nativeToken: eth,
   },
 };
+
+const MS_PER_DAY = 1000 * 60 * 60 * 24;
```

Comment:
We have [this file](https://github.com/VenusProtocol/venus-protocol-interface/blob/b1e40a4e380b7d6fac0cba0063ca347ef5f4dc05/apps/evm/src/constants/time.ts) that contains some time constants, you could add this one to it.